formstack-baton-requests/src/test/scala/com/gu/identity/formstackbatonrequests/FormstackPerformSarHandlerSpec.scala

```diff
@@ -0,0 +1,45 @@
+package com.gu.identity.formstackbatonrequests
+
+import com.gu.identity.formstackbatonrequests.aws.SubmissionTableUpdateDate
+import org.scalatest.{FreeSpec, Matchers}
+
+class FormstackPerformSarHandlerSpec extends FreeSpec with Matchers {
```

Comment:
I'll add more tests to this in next PR (which adds more functionality to FormstackPerformSarHandler)